### PR TITLE
Downgrade jobs with blocking presubmit coverage to non-blocking

### PR DESCRIPTION
**Pull request by fejta**
/assign @rmmh 

Since all blocking jobs have presubmit this effectively eliminates the blocking job list for now

https://docs.google.com/document/d/10AN_hgEM77CLtfZ-3v7zRV4ep5phqw2jc0jY4n8WRus/edit

### CODE REVIEW

**Comment by rmmh** [comment]
More accurately named "remove all blocking jobs".

Should we remove retest-not-required?

**Comment by fejta** [comment]
I do not think that is related as it has the result of not running the presubmit tests before merging.

**Comment by spxtr** [comment]
That's brave.

Note that because I don't think we ever plan on adding blocking ci jobs back, this does leave quite a lot of dead code in mungegithub.

**Comment by rmmh** [comment]
We can go through later to rip it out.

File: testgrid/config/config_test.go [hunk 1]
```diff
@@ -174,6 +174,9 @@ func TestConfig(t *testing.T) {
 
 	sqJobs := strings.Split(sqData.Data["submit-queue.jenkins-jobs"], ",")
 	for _, sqJob := range sqJobs {
+		if sqJob == "\"\"" { // ignore empty list of jobs
```
Comment (rmmh) [inline testgrid/config/config_test.go]: is this the right way to do it? Can we not just specify an empty key?

Comment (fejta) [inline testgrid/config/config_test.go]: I'm following the pattern used for other empty values like `submit-queue.required-contexts`.

https://github.com/kubernetes/test-infra/pull/2564/files#diff-8c98b46a995621a05311c6330e8926c3R33

I suspect there's some goofy bash reason why we want to do it this way.

**Pull request merged**